activesupport/lib/active_support/railtie.rb

```diff
@@ -9,6 +9,12 @@ class Railtie < Rails::Railtie # :nodoc:
 
     config.eager_load_namespaces << ActiveSupport
 
+    initializer "active_support.isolation_level" do |app|
```

Comment:
@byroot I think this needs an `after_initialize`, like other initializers in this railtie. See https://github.com/rails/rails/pull/42860 for example.

I can make a PR tomorrow, I need to run just now. But I quickly verified that this code gets called *before* the new framework defaults initializer, which isn't what you want.

Reply:
sounds good, I'll let you do the PR unless you don't have time.